akka-docs/rst/java/cluster-sharding.rst

```diff
@@ -208,14 +225,18 @@ The settings for Distributed Data is configured in the the section
 ``akka.cluster.sharding.distributed-data``. It's not possible to have different 
 ``distributed-data`` settings for different sharding entity types.
 
-You must explicitly add the ``akka-distributed-data`` dependency to your build if
-you use this mode. It is possible to remove ``akka-persistence`` dependency from a project if it
-is not used in user code.
+Persistence Mode
+^^^^^^^^^^^^^^^^
 
-.. warning::
+This mode is enabled with configuration::
+
+  akka.cluster.sharding.state-store-mode = persistence
+
+Since it is running in a cluster :ref:`persistence-java` must be configured with a distributed journal.
+
+You must explicitly add the ``akka-persistence`` dependency to your build if
```

Comment:
Add some note that it is very important to watch out for cluster splits and not use auto downing with it? Just a small sentence would be enough o think, no need for a note block